src/map.rs

```diff
@@ -1307,25 +1294,25 @@ where
         // there are three cases when a bin pointer is invalidated:
         //
         //  1. if the table was resized, bin is a move entry, and the resize has completed. in
-        //     that case, the table (and all its heads) will be dropped in the next epoch
-        //     following that.
+        //     that case, the table (and all its heads) will be retired.
         //  2. if the table is being resized, bin may be swapped with a move entry. the old bin
-        //     will then be dropped in the following epoch after that happens.
+        //     will be retired.
```

Comment:
Same here:

2. if the table is being resized, bin may be swapped with a move entry. the old bin will only be retired after that happens.

Reply:
Fixed in f24162f1abd3f954060eff5026e961a95a876231